Library/Sources/SCRecordSession.m

```diff
@@ -439,8 +439,7 @@ - (CMSampleBufferRef)adjustBuffer:(CMSampleBufferRef)sample withTimeOffset:(CMTi
         pInfo[i].duration = duration;
     }
     
-    CMSampleBufferRef sout;
-    CMSampleBufferCreateCopyWithNewTiming(nil, sample, count, pInfo, &sout);
+    CMSampleBufferRef sout = sample;
```

Comment:
Here you are simply ignoring the adjustment made to the buffer above. Why do you need to do that?